packages/video_player/video_player/lib/video_player.dart

```diff
@@ -355,9 +355,14 @@ class VideoPlayerController extends ValueNotifier<VideoPlayerValue> {
         break;
     }
 
-    if (videoPlayerOptions?.mixWithOthers != null) {
-      await _videoPlayerPlatform
-          .setMixWithOthers(videoPlayerOptions!.mixWithOthers);
+    if (videoPlayerOptions == null ||
+        (videoPlayerOptions != null &&
+            videoPlayerOptions!.bypassAudioSetup == false)) {
```

Comment:
This whole condition can simplify down to `!(videoPlayerOptions?.bypassAudioSetup ?? false)`.